updating links from fwlink to real link if exists (#16730)
dotnet/docs: docs/framework/wcf/feature-details/security-concepts-used-in-wcf.md

 WCF supports some of those infrastructures, such as Secure Sockets Layer (SSL) over HTTP (HTTPS). However, WCF goes beyond supporting existing security infrastructures by implementing newer interoperable security standards (such as WS-Security) over SOAP-encoded messages. Whether you are using existing mechanisms or new interoperable standards, the security concepts behind both are the same. Understanding the concepts behind existing infrastructures and the newer standards is central to implementing the best security model for an application.  
  
## Introduction to Security for WCF Web Services  

>  The Microsoft Patterns and Practices group wrote an in-depth white paper on WCF security guidance which is available for download here: [WCF Security Guide](https://archive.codeplex.com/?p=wcfsecurityguide). This white paper describes the fundamental security concepts as they relate to web services, key WCF security concepts, intranet application scenarios, and internet application scenarios.  

Reviewer: ```suggestion

The Microsoft Patterns and Practices group wrote an in-depth white paper called [WCF Security Guide](https://archive.codeplex.com/?p=wcfsecurityguide). This white paper describes the fundamental security concepts as they relate to web services, key WCF security concepts, intranet application scenarios, and internet application scenarios.  
```

  
## Industry-Wide Security Specifications  
  
### Public Key Infrastructure  

>  Public Key Infrastructure (PKI) is a system of digital certificates, certification authorities, and other registration authorities that verify and authenticate each party involved in an electronic transaction through the use of public key cryptography. For more information, see [Windows Server 2008 R2 Certificate Services](https://azure.microsoft.com/?ocid=cloudplat_hp).  

Reviewer: ```suggestion

Public Key Infrastructure (PKI) is a system of digital certificates, certification authorities, and other registration authorities that verify and authenticate each party involved in an electronic transaction through the use of public-key cryptography.
```
